tools/interactive/interactivetool_odv.xml

```diff
@@ -20,9 +20,11 @@
         mkdir -p Documents/ODV/galaxy/data &&
         chmod 777 Documents/ODV/ -R &&
         #if $method.type == 'own_data' :
-            #for $infile in $method.infiles:
-                yes | cp '$infile' './Documents/ODV/galaxy/data/$infile.display_name' &&
-            #end for
+            #for $i, $infile in enumerate($method.infiles):
+                #if $infile.ext == "netcdf":
+                    cp '$infile' './Documents/ODV/galaxy/data/${infile.element_identifier}_${i}.nc' &&
+                #end if
```

Comment:
Now you are not copying tabular files at all ...

Reply:
added an tested ! Sorry .. thanks !!